istio/src/main/protobuf/http_fault.proto

```diff
@@ -0,0 +1,104 @@
+//
+// Copyright 2017 Istio Authors
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+
+syntax = "proto3";
+
+import "duration.proto";
```

Comment:
in the istio/api repo i see this line as:
```
import "google/protobuf/duration.proto";
```

is it possible for us to conform to a directory structure such that the files will not require modification?

related: for the istio telemeter i'm going to need proto files under `api/mixer/v1/config`, can we move `http_fault.proto` and friends down into a directory structure that mirrors the api repo they came from?